libs/schema-profunctor/README.md

```diff
@@ -153,7 +153,7 @@ NonEmpty list, as there are no instances for `ToSchema [a]` or
 `nonEmptyArray` can be used to derive a schema for these if there are
 `ToSchema` instances for `UserId` and `Permission`.
 
-### Sum types
+### Sum types 
```

Comment:
this hasn't been fixed yet.  was i supposed to review this again?  i'll stop now!  :)